tox_conda/plugin.py

```diff
@@ -230,6 +246,8 @@ def tox_testenv_install_deps(venv, action):
     num_conda_deps = len(venv.envconfig.conda_deps)
     if venv.envconfig.conda_spec is not None:
         num_conda_deps += 1
+    if venv.envconfig.conda_env is not None:
+        num_conda_deps += 1
```

Comment:
At this point in the steps of tox, it seems to me that the information from `conda_env` has already been used to create the environment and it will not be used by `install_conda_deps`, so why is it being taken into account to determine if `install_conda_deps` shall be called?

Reply:
Right. It doesn't need to be taken into account for the check, but it needs to be taken into account for the removal of the "fake-dependencies" in line 257. I'll move it into the if statement.